enterprise/backend/src/metabase_enterprise/sso/integrations/jwt.clj

```diff
@@ -53,13 +55,24 @@
   "Registered claims in the JWT standard which we should not interpret as login attributes"
   [:iss :iat :sub :aud :exp :nbf :jti])
 
+(defn- filter-non-string-attributes
+  [jwt-data]
+  (->> jwt-data
+       (filter (fn [[key value]]
+                 (if (string? value)
+                   value
+                   (log/warnf "Dropping JWT claim '%s' with non-string value: %s" (name key) value))))
+       (into {})))
+
 (defn- jwt-data->login-attributes [jwt-data]
-  (apply dissoc
-         jwt-data
-         (jwt-attribute-email)
-         (jwt-attribute-firstname)
-         (jwt-attribute-lastname)
-         registered-claims))
+  (filter-non-string-attributes
+   (apply dissoc
+          jwt-data
+          (jwt-attribute-email)
+          (jwt-attribute-firstname)
+          (jwt-attribute-lastname)
+          (jwt-attribute-groups)
```

Comment:
Theoretically could someone already be doing something like sandboxing users based on the new attribute here?

Reply:
Alright, that makes sense. I had the impression that they were using it in some other way that did work (besides roles) but I think I was mistaken.